internal/log/glg/glg.go

```diff
@@ -81,6 +85,10 @@ func (l *logger) Infof(format string, vals ...interface{}) {
 	l.retry.Outf(l.glg.Infof, format, vals...)
 }
 
+func (l *logger) Infod(msg string, details ...interface{}) {
+	l.Infof(msg+"\tdetails: %#v", details)
```

Comment:
Is this format okay? :thinking:

Reply:
please define default details format as const value on the top

Reply:
done.